src/dtscalibration/calibration/section_utils.py

```diff
@@ -0,0 +1,223 @@
+from typing import Dict
+from typing import List
+
+import numpy as np
+import xarray as xr
+import yaml
+
+from dtscalibration.datastore_utils import ufunc_per_section_helper
+
+
+def set_sections(ds: xr.Dataset, sections: Dict[str, List[slice]]) -> xr.Dataset:
```

Comment:
Maybe change the name to set_sections_to_ds() to better describe what it does.

Reply:
That feels a bit redundant to me. It's a function that takes a dataset and sections, and returns a dataset.

Reply:
Ok! Let's merge